### PR TITLE
changed research dark mode #988

### PR DESCRIPTION
**Pull request by nik464**
I have  changed dark mode of research page. Gave purple colour with box shadow . Button has also got same colour with shadow as mentioned in task.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -880,7 +877,7 @@ form {
 
 .form-style-6 input[type="submit"]:hover,
 .form-style-6 input[type="button"]:hover {
-    background:#6a00bbd2;;
+    background:#80498a;;
```
Comment (smaranjitghose) [inline assets/css/style.css:880]: Why is this changed?

**Comment by smaranjitghose** [comment]
Path

The changes do not conform to the requirements at all. Thus we cannot consider this PR

**Comment by nik464** [comment]
we did nt have to change color to purple scheme in dark mode and apply the same to button as well or something has been missed?
